addon/components/file-dropzone/component.js

```diff
@@ -68,14 +68,16 @@ const dragListener = new DragListener();
   @yield {Hash} dropzone
   @yield {boolean} dropzone.supported
   @yield {boolean} dropzone.active
-  @yield {valid} dropzone.valid
+  @yield {boolean} dropzone.valid
   @yield {Queue} queue
  */
 export default BaseComponent.extend({
 
   layout,
 
   supported,
+  active: false,
+  valid: true,
```

Comment:
These properties were initially undefined. I think this was a violation of the public API (boolean). Also nice to have them explicitly defined here.